src/main/java/org/gridsuite/actions/server/ContingencyListService.java

```diff
@@ -252,20 +280,36 @@ public ScriptContingencyList newScriptFromFormContingencyList(UUID id, UUID newI
         });
     }
 
-    private static IdBasedContingencyList fromIdBasedContingencyListEntity(IdBasedContingencyListEntity entity) {
+    private static IdBasedContingencyList fromIdBasedContingencyListEntity(IdBasedContingencyListEntity entity, Network network) {
         List<NetworkElementIdentifier> listOfNetworkElementIdentifierList = new ArrayList<>();
+        Map<String, Set<String>> notFoundElements = new HashMap<>();
         entity.getIdentifiersListEntities().forEach(identifierList -> {
             List<NetworkElementIdentifier> networkElementIdentifiers = new ArrayList<>();
-            identifierList.getEquipmentIds().forEach(equipmentId -> networkElementIdentifiers.add(new IdBasedNetworkElementIdentifier(equipmentId)));
+            identifierList.getEquipmentIds().forEach(equipmentId -> {
+                if (network != null && network.getIdentifiable(equipmentId) == null) {
+                    Set<String> ids = notFoundElements.get(identifierList.getName());
```

Comment:
Use Map.computeIfAbsent to be more concise ?

Reply:
Done